File: Gorev_3/JavaScript/main.js

const getLocation = async () => {
    const ip = document.getElementById("ip").value
    await axios.get(`http://ip-api.com/json/${ip}`)
    .then(function (res) {
        if(res.data.status === 'fail'){
            throw 'IP Adresi Geçerli Değil'
        }
        let out = `ip: ${res.data.query}\nlatitude: ${res.data.lat}\nlongitude: ${res.data.lon}\ncountry: ${res.data.country}`;
        document.getElementById("result").innerText = `${out}`;
        console.log(res);
    })
    .catch(function (res) {
        // handle error
        document.getElementById("result").innerText = `${res}`;
    })
    .then(function () {
        // always executed
    });
}

File: Gorev_3/Python/main.py
import requests
def main(ip):
    req = requests.get(f'http://ip-api.com/json/{ip}')
    res = req.json()
    try:
        return f'ip: {res["query"]}\nlatitude: {res["lat"]}\nlongitude: {res["lon"]}\ncountry: {res["country"]}'
    except:
        return f'Status: {res["status"]}! Message: {res["message"]}'

if __name__ == '__main__':
    inp = input('IP Adres Giriniz: ')
    print(main(inp))